mets_mods2tei/api/mets.py

```diff
@@ -227,16 +227,16 @@ def __spur(self):
         if amdsec and amdsec[0].get_rightsMD():
             dv = etree.fromstring(amdsec[0].get_rightsMD()[0].get_mdWrap().get_xmlData().get_anytypeobjs_()[0])
         else:
-            dv = []
+            dv = None
 
         # owner of the digital edition
-        self.owner_digital = dv.xpath("//dv:owner", namespaces=ns)[0].text if dv else ""
+        self.owner_digital = dv.xpath("//dv:owner", namespaces=ns)[0].text if dv is not None else ""
```

Comment:
Is this variant more pythonic?

Reply:
It's not IMHO – but that's unfortunately what newer Python versions now require. (You'll even get a warning for not using `len()` tests...)